pkg/logs/sender/config.go

```diff
@@ -40,7 +40,7 @@ func BuildEndpoints() (*client.Endpoints, error) {
 		ProxyAddress: proxyAddress,
 	}
 	switch {
-	case config.Datadog.IsSet("logs_config.logs_dd_url"):
+	case config.Datadog.IsSet("logs_config.logs_dd_url") && config.Datadog.GetString("logs_config.logs_dd_url") != "":
```

Comment:
could we create a function that wraps this logic ?

Reply:
Done and unit test